packages/ui/src/common/email-field.tsx

```diff
@@ -50,7 +70,7 @@ export function EmailField({
         </Common.FieldState>
         <Common.FieldError asChild>
           {({ message, code }) => {
-            return <Field.Message intent='error'>{error ? error(message, code, ERROR_NAME) : message}</Field.Message>;
+            return <Field.Message intent='error'>{translateError(message, code, DEFAULT_ERROR_NAME)}</Field.Message>;
```

Comment:
Can easily enable a custom translateError prop if/when its needed in the future.

Reply:
add for backwards compat for the time being https://github.com/clerk/javascript/pull/3615/commits/476a0656df6f4a43cb450b5fac79356c0d4b5dac